src/main/java/gregtech/common/metatileentities/converter/ConverterTrait.java

```diff
@@ -251,12 +251,12 @@ public int extractEnergy(int maxExtract, boolean simulate) {
 
         @Override
         public int getEnergyStored() {
-            return FeCompat.toFe(storedEU, FeCompat.ratio(feToEu));
+            return (int) Math.min(Integer.MAX_VALUE, FeCompat.toFeLong(storedEU, FeCompat.ratio(feToEu)));
```

Comment:
You could move this math to `FeCompat` and have function like `toFeIntLimited` with documentation why it is needed. As not everyone will be able to think about that problem.